tdrs-backend/wait_for_postgres.py

```diff
@@ -24,7 +24,7 @@
 
 def pg_isready(host, user, password, dbname):
     """Check to see if Postgres is ready."""
-    while time() - start_time < check_timeout:
+    while time() - start_time < float(check_timeout):
```

Comment:
This is an unrelated fix for a local development error that occurs when using the `POSTGRES_CHECK_TIMEOUT` env var.